docs/installation_configuration_guide/model/Lite.md

```diff
@@ -2,22 +2,22 @@
 sidebar_label: "Lite"
 ---
 
-# Streaming-Lite specific model configuration
+# Lite Model configuration
 
+## Streaming processing mode
```

Comment:
Do we have non-streaming mode here?

Reply:
Right now we do not have and this subsection is deliberately added to be consistent